agent/config/pbench-agent-default.cfg

```diff
@@ -65,3 +65,8 @@ host_remote_user = root
 stockpile_path =
 stockpile_log =
 stockpile_output_path =
+
+[container-registry]
+pcp_pmlogger = quay.io/meyceoz/pcp-collector
+pcp_pmcd = quay.io/meyceoz/pcp-exposer
```

Comment:
For the PCP entries above, `s/meyceoz/pbench/`, and then add the Dockerfile for this collector to our `agent/containers/images` directory?